app/views/foreman_leapp/job_templates/leapp_install.erb

```diff
@@ -0,0 +1,35 @@
+<%#
+kind: job_template
+name: Install Leapp
+job_category: Leapp - Install
+description_format: 'Install Leapp CI tool'
+provider_type: Ansible
+model: JobTemplate
+oses:
+- RedHat
+%>
+<%
+  os_major = @host.operatingsystem.major.to_i
+  rhel_compatible = @host.operatingsystem.family == 'Redhat' && @host.operatingsystem.name == 'RedHat'
+  repos = os_major == 7 ? ['rhel-7-server-rpms', 'rhel-7-server-extras-rpms'] :
+                          ['rhel-8-for-x86_64-baseos-rpms', 'rhel-8-for-x86_64-appstream-rpms']
+
+  package = os_major == 7 ? 'leapp' : 'leapp-upgrade'
+-%>
+---
+- hosts: all
+  tasks:
+<% if rhel_compatible -%>
+<% repos.each do |repo| -%>
+    - name: Enable <%= repo %>
+      command: subscription-manager repos --enable <%= repo %>
+<% end -%>
```

Comment:
Wouldn't it be better to enable all the repositories with a single call to sub-man?